matchms/exporting/save_as_mzspeclib.py

```diff
@@ -0,0 +1,201 @@
+import os
+import re
+from typing import List, TextIO
+from matchms.Spectrum import Spectrum
+
+
+ANALYTE_ATTRIBUTES = {
+    'formula': 'MS:1000866|molecular formula',
+    'smiles': 'MS:1000868|SMILES formula',
+    'inchi': 'MS:1003403|InChI',
+    'inchikey': 'MS:1002894|InChIKey'
+}
+
+SPECTRUM_ATTRIBUTES = {
+    'compound_name': 'MS:1003061|library spectrum name',
+    'precursor_mz': 'MS:1003208|experimental precursor monoisotopic m/z',
+    'scans': 'MS:1003057|scan number',
+    'charge': 'MS:1000041|charge state',
+    'ionmode': 'MS:1000465|scan polarity'
```

Comment:
You could maybe add retention_time, which is stored as "retention_time" by add_retention_time filter. (but maybe this is not easy, since we don't support the s/min labeling, in that case ignore this comment.)

Reply:
Yeah we don't know which unit it is in and I'd rather not write it out than write it out wrong